meow/frontend/src/components/EditPost/EditForm.css

```diff
@@ -44,3 +44,11 @@
   right: 0px;
   color: red;
 }
+.no-section-error {
+  color: red;
+  font-size: small;
+}
+.no-url-error {
```

Comment:
For this one, because its just a warning and shouldn't prevent the person from click "Ready to Publish," it would be better if it was some kind of yellow color. Yellow is generally asscociated with warning. I would recommend the yellow from [here](https://ant.design/docs/spec/colors#Functional-Color). But if this yellow doesn't look good, feel free to pick another yellow that looks good.